lib/features/modeling/behavior/SetCompensationActivityAfterPasteBehavior.js

```diff
@@ -48,7 +48,7 @@ SetCompensationActivityAfterPasteBehavior.$inject = [
 
 function isForCompensation(element) {
   const bo = getBusinessObject(element);
-  return bo && typeof bo.get === 'function' && bo.get('isForCompensation');
+  return bo && is(bo, 'bpmn:Task') && bo.get('isForCompensation');
```

Comment:
Q: What other cases do you guard against (protect against)? Any business object that is a `moddle element` has a `get` method.

Reply:
Some test cases for `BpmnCreateMoveSnapping` involving `LabelImp` were failing because, while it can fetch the `businessObject`, it cannot call `bo.get` on it.

Reply:
I'll investigate this, thanks for the heads-up.

Reply:
Merging changes.